src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -785,7 +785,7 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                     )}
                 >
                     <button
-                        className={`cta flex h-36 ${this.isDeployButtonDisabled() ? 'disabled-opacity' : ''}`}
+                        className={`cta flex ml-auto h-36 ${this.isDeployButtonDisabled() ? 'disabled-opacity' : ''}`}
```

Comment:
Why did you add margin left auto, did you check without this? Was it not working as expected?

Reply:
Trigger button was to be on right , added ml-auto to post it on right .

Reply:
okay got it